crates/re_renderer/Cargo.toml

```diff
@@ -95,6 +95,8 @@ zip = { workspace = true, default-features = false, features = ["deflate"] }
 unindent.workspace = true
 
 # native
+
+
```

Comment:
??

Reply:
`taplo fmt` does this. I don't know why.

Reply:
I tried to reformat it to make `taplo` happy